docs/tutorials/integrations/authenticators.md

```diff
@@ -48,13 +48,8 @@ irb(main):001:0> require 'slosilo'
 => true
 irb(main):002:0> key = Slosilo::Key.new
 => #<Slosilo::Key:0x00000001999098 @key=#<OpenSSL::PKey::RSA:0x00000001999048>>
-irb(main):003:0> puts JSON.pretty_generate(key.signed_token('alice'))
-{
-  "data": "alice",
-  "timestamp": "2017-06-14 15:14:04 UTC",
-  "signature": "AA2x...rFnF",
-  "key": "8485e4f593dd2668a062cdaecf28d5bd"
-}
+irb(main):003:0> puts key.issue_jwt(sub: 'alice').to_json
+{"protected":"eyJhbGciOiJjb25qdXIub3JnL3Nsb3NpbG8vdjIiLCJraWQiOiIzZTY4N2E3N2Q0ZjkzOTkxYzZmMzBkMzkzYTNmZGM1MyJ9","payload":"eyJzdWIiOiJhbGljZSIsImlhdCI6MTUwNTgzMjg1NX0=","signature":"jzwY1MmbYQUElR8qA8mFOhWqb2G96W1uaB_BrrMnhUZzNMqVv0g6Z93zjD_KQn4mEOwwV9JcDopsiksvzpIpFsxulE9VtimEQhVoZjBDUmmLCvgjg0feX5YFSCMxHmgZsVs5azwNi8f51URmuIDzRRzJ2AaNWjbNpkx3MSrpjDgRpqbRL1ryVTGkBb7rnYSYhHc8fSK-jRpXrJCFZIf1cSMNKcHXJ3bvSjgxnX4Jv4AfTSahQ9wXseDWibF7tqDfABrewsptKfacJCMGku0OFGcxdUMV0ajgDnflf5kRXHY7UW8-3cXPzQO43-kgeTPIuHedrKfuZnGBA4k8TYnNIz56pNW_e9xzXp13vNYAaquHcTXc_jJeus-6l2OyHzDx"}
```

Comment:
What do you think about formatting the returned JSON for readability?  A single line doesn't feel like the most digestible format (although probably the most accurate).  Maybe something like:
```json
{
  "protected":"eyJhbGciOiJjb25qdXIub3JnL3Nsb3NpbG8vdjIiLCJraWQiOiIzZTY4N2E3N2Q0ZjkzOTkxYzZmMzBkMzkzYTNmZGM1MyJ9",
  "payload":"eyJzdWIiOiJhbGljZSIsImlhdCI6MTUwNTgzMjg1NX0=",
  "signature":"jzwY1MmbYQUElR8qA8mFOhWqb2G96W1uaB_BrrMnhUZzNMqVv0g6Z93zjD_KQn4mEOwwV9JcDopsiksvzpIpFsxulE9VtimEQhVoZjBDUmmLCvgjg0feX5YFSCMxHmgZsVs5azwNi8f51URmuIDzRRzJ2AaNWjbNpkx3MSrpjDgRpqbRL1ryVTGkBb7rnYSYhHc8fSK-jRpXrJCFZIf1cSMNKcHXJ3bvSjgxnX4Jv4AfTSahQ9wXseDWibF7tqDfABrewsptKfacJCMGku0OFGcxdUMV0ajgDnflf5kRXHY7UW8-3cXPzQO43-kgeTPIuHedrKfuZnGBA4k8TYnNIz56pNW_e9xzXp13vNYAaquHcTXc_jJeus-6l2OyHzDx"
}
```